src/js/components/TextInput/TextInput.js

```diff
@@ -129,6 +132,25 @@ const TextInput = forwardRef(
     );
 
     const [suggestionsAtClose, setSuggestionsAtClose] = useState();
+    const [tipContent, setTipContent] = useState(
+      format({ id: 'input.readOnlyCopyValidation', messages }),
+    );
+
+    const copyOnClick = () => {
```

Comment:
```suggestion
    const onClickCopy = () => {
```
small naming suggestion for handler to start with `on`

Reply:
Good idea, changed in latest commit